package.json

```
@@ -3999,6 +3999,446 @@
+														"type": "command",
+														"command": "editor.action.goToReferences"
+													},
+													{
+														"key": "s",
+														"name": "Go to symbol in file",
+														"icon": "file",
+														"type": "command",
+														"command": "workbench.action.gotoSymbol"
+													},
+													{
+														"key": "R",
+														"name": "Find references",
+														"icon": "symbol-reference",
+														"type": "command",
+														"command": "references-view.findReferences"
+													},
+													{
+														"key": "S",
+														"name": "Go to symbol in project",
+														"icon": "project",
+														"type": "command",
+														"command": "workbench.action.showAllSymbols"
+													}
+												]
+											},
+											{
+												"key": "m",
+												"name": "+Miscellaneous",
+												"icon": "symbol-property",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "c",
+														"name": "Change directory here",
+														"icon": "folder-opened",
+														"type": "command",
+														"command": "language-julia.cdHere"
+													},
+													{
+														"key": "d",
+														"name": "Clear сurrent inline results",
+														"icon": "chrome-close",
+														"type": "command",
+														"command": "language-julia.clearCurrentInlineResult"
+													},
+													{
+														"key": "D",
+														"name": "Clear all inline results",
+														"icon": "clear-all",
+														"type": "command",
+														"command": "language-julia.clearAllInlineResults"
+													},
+													{
+														"key": "K",
+														"name": "Clear all inline results in editor",
+														"icon": "clear-all",
+														"type": "command",
+														"command": "language-julia.clearAllInlineResultsInEditor"
+													},
+												]
+											},
+											{
+												"key": "s",
+												"name": "+REPL",
+												"icon": "repl",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "i",
+														"name": "Start REPL",
+														"icon": "repl",
+														"type": "command",
+														"command": "language-julia.startREPL"
+													},
+													{
+														"key": "D",
+														"name": "Stop REPL",
+														"icon": "stop",
+														"type": "command",
+														"command": "language-julia.stopREPL"
+													},
+													{
+														"key": "C",
+														"name": "Connect external REPL",
+														"icon": "vm-connect",
+														"type": "command",
+														"command": "language-julia.connectREPL"
+													},
+													{
+														"key": "s",
+														"name": "Execute code in REPL (ctrl+Enter)",
```

```suggestion
														"name": "Execute code in REPL",
```

I would suggest to remove the parenthesis as it might be confusing between the key `s` and `Ctrl+Enter` and instead put those extra keys as documentation in https://vspacecode.github.io/docs/major-mode.

---

My reason for putting them is is that these are super common commands, and I like how it gives a reminder on a faster way to do it. It is clunky, though, so I'll remove it. I like how the normal VS Code command menu shows the keys. I'll open an issue on that.

---

We will address that in the separate thread. Thanks for updating.